site/src/pages/_app.page.tsx

```diff
@@ -1,4 +1,3 @@
-/** @sync ../components/Snippet.tsx */
```

Comment:
Bonus: removed stale comment (backdates to https://github.com/blockprotocol/blockprotocol/pull/2)